dev-python/pybedtools/pybedtools-0.10.0.ebuild

```diff
@@ -24,8 +24,9 @@ RDEPEND="
 	dev-python/pandas[${PYTHON_USEDEP}]
 	dev-python/matplotlib[${PYTHON_USEDEP}]
 "
-
-BDEPEND="dev-python/cython[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/cython[${PYTHON_USEDEP}]
+"
```

Comment:
This is not necessary, cython dep is added by `DISTUTILS_EXT=1` (and it is definitely a bdep not dep)